modules/addRemove.js

```diff
@@ -93,6 +102,49 @@ const wholeList = function () {
     }
   });
 
+  // Function to check box
+  list.addEventListener('change', (e) => {
+    const target = e.target.closest('.to-do-input');
+    const text = e.target.nextElementSibling;
+    const targetID = Number(e.target.id);
+
+    if (!target) return;
+    if (target.checked) {
+      text.style.textDecoration = 'line-through';
+      getStorage[targetID].completed = true;
+      setStorage();
+    } else {
+      text.style.textDecoration = 'none';
+      getStorage[targetID].completed = false;
+      setStorage();
+    }
+  });
+
+  const domLoaded = () => {
+    const text = document.querySelectorAll('.to-do-text');
+    const checkbox = [...document.querySelectorAll('.to-do-input')];
+    getStorage.forEach((item, i) => {
+      if (item.completed === true) {
+        checkbox[i].checked = true;
+        text[i].style.textDecoration = 'line-through';
+      }
+    });
+  };
```

Comment:
- [ ] As demanded by project requirements, kindly implement this function in a new file and import it as a module. This is best ES6 practice.